workflow/scripts/build_renewable_profiles.py

```diff
@@ -226,7 +226,8 @@ def plot_data(data):
     cutout = atlite.Cutout(snakemake.input.cutout).sel(time=sns)
 
     regions = gpd.read_file(snakemake.input.regions)
-    assert not regions.empty, (
+
+    assert "x" in regions.columns and "y" in regions.columns, (
         f"List of regions in {snakemake.input.regions} is empty, please "
```

Comment:
This caused the following error to appear when running the default configuration file. I had to remove the `offwind` and `offwind_floating` techs to get it to work. If this is required, we should probably just remove them from the default config, or (ideally?) correct this issue?

Reply:
created issue #426